samples/apps/spring-petclinic/.minikube/deploy-app.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+
+NAMESPACE=${NAMESPACE:-my-postgresql}
+
+sed -e 's,REPLACE_PETCLINIC_APP_IMAGE,'${PETCLINIC_APP_IMAGE}',g' app-deployment.yaml | \
+sed -e 's,REPLACE_PETCLINIC_APP_DBINIT_IMAGE,'${PETCLINIC_APP_DBINIT_IMAGE}',g' | \
+kubectl apply -f - -n ${NAMESPACE} --wait
+kubectl delete service spring-petclinic --ignore-not-found=true -n ${NAMESPACE}
```

Comment:
why is this needed?

Reply:
this was to be able to run that multiple times for the developement... if you try to expose a deployement as a service that already exists, it would fail... but I changed that - I included the service in the app-deployment.yaml where it is applied, rather then created.

Reply:
so no need for the deletion - removed